workflows/prognostic_c48_run/runtime/derived_state.py

```diff
@@ -22,7 +22,11 @@ class FV3StateMapper(Mapping):
 
     def __init__(self, getter, alternate_keys: Mapping[str, str] = None):
         self._getter = getter
-        self._alternate_keys = alternate_keys or {"lon": "longitude", "lat": "latitude"}
+        self._alternate_keys = alternate_keys or {
```

Comment:
Can we do without the renaming or maybe pass it via the `alternate_keys` argument?

Reply:
The `physics_precip` name is hardcoded in the `PrecipitativeModel` and I think using that name in there (as opposed to `surface_precipitation_rate` does help clarify how the architecture works. 

By 'pass via the `alternate_keys` arg`, do you mean pass it in when the loop's `state` is initialized in `loop.py`? If so, what's the benefit in doing that versus having it be in the default `_alternate_keys`? Just wondering since the "physics_precip"/"surface_precipitation_rate" pair is hard coded into the base diagnostics that get saved, so I assumed it'd be an ok alternate key pair to include here.

Reply:
> Do you mean pass it in when the loop's stateis initialized inloop.py

That's right. Best to be up-front with this kind of "default" behavior IMO rather than hiding it in the code.

However, I just remembered the  `runtime.machine_learning.RenamingAdapter` also provides a configurable renaming via `config.scikit_learn.input_standard_names`. It doesn't seem like any code needs to change to support this renaming.

Reply:
As discussed in person, I'll leave this here since the renaming pair is kind of buried in the prog run code, which can be a pain to dig out when writing the config (ex. I renamed it to the wrong precip variable the first time I tested a model using  this input)